src/redux/books/books.js

```diff
@@ -0,0 +1,50 @@
+import { createReducer, createAction } from '@reduxjs/toolkit';
+
+export const ADD_BOOK = createAction('books/add');
+export const REMOVE_BOOK = createAction('book/remove');
+
+const initialState = {
+  books: [
+    {
+      id: '1',
+      title: 'The Hunger Games',
+      author: 'Suzanne Collins',
+      category: 'Action',
+    },
+    {
+      id: '2',
+      title: 'Dune',
+      author: 'Frank Herbert',
+      category: 'Science Fiction',
+    },
+    {
+      id: '3',
+      title: 'Capital in the Twenty-First Century',
+      author: 'Suzanne Collins',
+      category: 'Economy',
+    },
+  ],
+};
+
+const booksReducer = createReducer(initialState, (builder) => {
+  builder.addCase(ADD_BOOK, (state, action) => {
+    const updatedState = {
+      ...state,
+      books: [
+        ...state.books,
+        { ...action.payload, id: `${state.books.length + 1}` },
+      ],
+    };
+    return updatedState;
+  });
+
+  builder.addCase(REMOVE_BOOK, (state, action) => {
+    const updatedBooks = [...state.books].filter(
+      (book) => book.id !== action.payload,
+    );
+    return { ...state, books: updatedBooks };
+  });
+  builder.addDefaultCase((state) => state);
+});
```

Comment:
- [ ] It is possible to remove books from your application but reappear after page reload. Please look into that too.

[remove-book.webm](https://user-images.githubusercontent.com/26597432/215263250-737e43a4-f8cc-463c-9eb7-b5f03b9690e2.webm)

Reply:
Fixed.